client/src/components/UIElements/Loading.scss

```diff
@@ -0,0 +1,33 @@
+.loading-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    gap: 10px;
+    opacity: 0.2;
+    padding: 100px;
+}
+
+.loading-text {
+    font-weight: 600;
+    font-size: 1.2em;
+    color: var(--main-blue);
+}
+
+.loading-spinner{
+    width:40px;
+    height:40px;
+    border:4px solid var(--main-blue);
+    border-radius:50%;
+    border-top-color:rgba(168, 0, 0, 0.067);
+    display:inline-block;
+    animation:loadingspinner .7s linear infinite;
+  }
+  @keyframes loadingspinner{
+    0%{
+      transform:rotate(0deg)
+    }
+    100%{
+      transform:rotate(360deg)
+    }
+  }
```

Comment:
You can make this styles safer buy putting them inside ".loading-container { ... }"